app/questionnaire/dependencies.py

```diff
@@ -55,3 +52,28 @@ def get_block_ids_for_calculated_summary_dependencies(
             block_ids_by_section[key] = routing_path.block_ids
 
     return block_ids_by_section
+
+
+def get_calculated_summary_block_ids_of_dependent_section(
```

Comment:
similar comment I think this doesn't indicate that its returning a map and a name adjustment would be helpful

Reply:
Updated to `map_section_ids_to_routing_path`.